src/ErrorCodes.js

```diff
@@ -0,0 +1,50 @@
+/*
+This file is part of kaya.
+Copyright (c) 2018 - present Zilliqa Research Pvt. Ltd.
+
+kaya is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later
+version.
+
+kaya is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+kaya.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* Configuration file */
+/* Feel free to add more things to this file that will help you in development */
+
+// Source: https://github.com/Zilliqa/Zilliqa/blob/master/src/libServer/Server.h 
+module.exports = {
```

Comment:
I would suggest using the version exported by `@zilliqa-js/core`, so that you'll get any upstream updates for free.